great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -176,36 +174,10 @@ def get_partitioner_implementation(
             )
         return PartitionerClass(**abstract_partitioner.dict())
 
-    @property
-    @override
-    def batch_request_options(
```

Comment:
Do we have a ticket to add this property to BatchConfigs and have them call this method on their asset?

Reply:
[v1-205](https://greatexpectations.atlassian.net/browse/V1-205) 👍